nextflow.config

```diff
@@ -43,6 +43,7 @@ params {
     // Cellranger parameters
     skip_cellranger_renaming = false
     cellranger_index         = null
+    cellranger_create_bam    = true
```

Comment:
What do you think of unifying this into a single parameter that supports all aligners that create bams (i.e. the different cellrangers and starsolo)?

E.g. in `#rnaseq` this is called `--save_align_intermeds`

Reply:
Happy to unify these flags into a single parameter for those aligners that create bams.  I'll review the `rnaseq` pipeline to get a better idea about how this was implemented.  Thank you again for your direction and support on this pull request.

Reply:
How does this look [36eeee](https://github.com/nf-core/scrnaseq/pull/387/commits/36eeee39339446e84a4d5362ff47562ff9fe5441)?

Reply:
Now is this parameter called `--create-bam` or `--save_align_intermeds`? Either is fine with me, but it should be consistent everywhere. 

Other than that, this is exactly what I had in mind. Can you implement the flag also for cellranger multi and cellranger arc?

Reply:
Hi @grst, I implemented the changes to cellranger multi, but not cellranger arc due to [#290](https://github.com/nf-core/scrnaseq/issues/290).  Thoughts?

Reply:
fair enough for cellranger-arc, I can't see anything related to cellranger multi though?

Reply:
This logic is currently being handled in `workflows/scrnaseq.nf` [36eeee](https://github.com/nf-core/scrnaseq/pull/387/commits/36eeee39339446e84a4d5362ff47562ff9fe5441#diff-1b4d9315e66b5c91a661df38866c6b1371117f443c1ada9d1a41b1cdcda2944a).

Reply:
Ah, sorry I missed that!